dxr/static/css/icons.css

```diff
@@ -85,6 +85,9 @@ a.type {
 }
 /* End of Panel icons that are always shown */
 
+.icon-container.folder, .promoted-icon.folder {
```

Comment:
I can't get icons to show up for promoted results, no matter how much I recompile the templates.